geoapps/driver_base/driver.py

```diff
@@ -97,6 +99,14 @@ def start(cls, filepath: str | Path, driver_class=None):
             filepath, validations=driver_class._validations  # pylint: disable=W0212
```

Comment:
**suggestion (readability):** use pylint name instead of numbers:  `protected-access`

or better: fix the warning rather than disabling it

Reply:
That's old stuff and going to be deprecated. I ll just change the pylint name for now